jsapp/js/components/permissions/sharingForm.component.tsx

```diff
@@ -81,6 +81,8 @@ export default class SharingForm extends React.Component<SharingFormProps, Shari
     )
     if (this.props.assetUid) {
       actions.resources.loadAsset({ id: this.props.assetUid })
+      // Update permissions on modal open
+      actions.permissions.getAssetPermissions(this.props.assetUid)
```

Comment:
AFAIK the thing we get from `actions.permissions.getAssetPermissions` (which is using `/api/v2/assets/<uid>/permission-assignments/` endpoint) is the same thing we get from `actions.resources.loadAsset` (which is using `/api/v2/assets/<uid>/`) inside `permissions` prop.

Reply:
Does it mean that this bug happens because of the `assetCache` from `actions.js`? I.e. we are getting cached asset after permissions were changed

Reply:
Maybe a bit better fix would be `actions.resources.loadAsset({ id: this.props.assetUid }, true)`?